src/CommonFunctions.js

```diff
@@ -540,6 +542,19 @@ const csvToCodes = (csv) => {
   }).filter(code => code.code !== null);
 };
 
+const determineTweetType = (tweet) => {
+  if (tweet.in_reply_to_status_id != null) {
```

Comment:
we should be able to treat these as truthy, so
```js
if (tweet.in_reply_to_status_id) {
```
instead of
```js
if (tweet.in_reply_to_status_id != null) {
```